documentation/CAMARA-Security-Interoperability.md

```diff
@@ -279,6 +290,175 @@ Camara recommends that implementations run the OIDF interoperability suite and a
 * [RFC 7519 - JSON Web Token (JWT)](https://www.rfc-editor.org/info/rfc7519)
 * [RFC 7521 - Assertion Framework for OAuth 2.0 Client Authentication and Authorization Grants](https://www.rfc-editor.org/info/rfc7521)
 * [RFC 7523 - JSON Web Token (JWT) Profile for OAuth 2.0 Client Authentication and Authorization Grants](https://www.rfc-editor.org/info/rfc7523)
+* [RFC 7636 - Proof Key for Code Exchange by OAuth Public Clients](https://www.rfc-editor.org/info/rfc7636)
 * [RFC 8259 - The JavaScript Object Notation (JSON) Data Interchange Format](https://www.rfc-editor.org/info/rfc8259)
 * [RFC 8414 - OAuth 2.0 Authorization Server Metadata](https://www.rfc-editor.org/info/rfc8414)
 
+## Appendix A: Error Responses
+
+This section describes the error responses that the Authorization Server MUST return for all flows defined in this profile. Based on the supported functionality, the set of errors and the scenarios that can generate them are defined from the specifications already referenced in this document.
+
+### Authentication Error Response
+
+#### Authorization Code Flow
```

Comment:
I think this new section is valuable, particularly for implementor technical teams we've spoken to that are not so close to OpenID specification, and for integrators who can use it as a concise requirements guide. I think it would be helpful if the sub-sections linked out to the relevant OIDC Error specification for readers to review the full context of the expected error response (e.g. [Authentication Error Response](https://openid.net/specs/openid-connect-core-1_0.html#AuthError)). For example, in Auth Code flow, there are other parameters that can be returned (e.g. a state parameter may be required depending on the input).

Reply:
I added new links to relevant error specifications.